src/mumble/MainWindow.cpp

```diff
@@ -1682,6 +1682,8 @@ void MainWindow::on_qaUserLocalMute_triggered() {
 	p->setLocalMute(muted);
 	if (! p->qsHash.isEmpty())
 		g.db->setLocalMuted(p->qsHash, muted);
+	else
+		msgBox(MainWindow::tr("This change could not be saved permanently and is lost on restart because %1 does not have a certificate.").arg(Log::formatClientUser(p, Log::Target)));
```

Comment:
`tr` is a static function of `QObject`, so you should access it via `QObject` like so:
```suggestion
		msgBox(QObject::tr("This change could not be saved permanently and is lost on restart because %1 does not have a certificate.").arg(Log::formatClientUser(p, Log::Target)));
```